internal/common/test_configuration.go

```diff
@@ -110,9 +110,10 @@ var TestImageStatusesFailure = &models.ContainerImageAvailability{
 	Result: models.ContainerImageAvailabilityResultFailure,
 }
 
-var DomainAPI = "api.test.example.com"
-var DomainAPIInternal = "api-int.test.example.com"
-var DomainApps = fmt.Sprintf("%s.apps.test.example.com", constants.AppsSubDomainNameHostDNSValidation)
+var DomainAPI = "api.test-cluster.example.com"
```

Comment:
I think it will be more clear if we use `example.invalid` as baseDnsDomain (in the test).

See https://github.com/openshift/assisted-service/blob/dd818a5650a66ff6693f632ddb83d48d29692755/internal/network/manifests_generator.go#L343

Reply:
All the DNS validation runs with the same resolution result so the cluster name+domain should be the same for all 4 resolved entries or else the unit tests will fail, and I think this name IS less fitting to the normal DNS validation so I will leave it with "example.com".
Also for the subsystem test, 99% defined with "example.com" in the test files already so if I change the name they will fail as well.